Tests/Amr/Advection_AmrLevel/CMakeLists.txt

```diff
@@ -1,8 +1,15 @@
-if ( (AMReX_SPACEDIM EQUAL 1) OR NOT CMAKE_Fortran_COMPILER_LOADED )
-   return()
+if (AMReX_SPACEDIM EQUAL 1)
+        message([WARNING] "Advection_AmrLevel Tests require AMReX_SPACEDIM = 2 or 3." )
```

Comment:
I don't think we should print a warning message here.  Users will see the warning if they do `-DAMReX_ENABLE_TESTS:BOOL=ON -DAMReX_SPACEDIM=1` when building amrex.  The message seems to imply there is something wrong, but that's not the case.

Reply:
If you are worried about that, it can be explained in the README file in that test.

Reply:
Got it. I removed the warning and put a short explanation in the README file.